tests/js/spec/views/performance/transactionSummary/transactionEvents/eventsTable.spec.tsx

```diff
@@ -14,6 +14,48 @@ type Data = {
   features?: string[];
 };
 
+export const mockEventsTableData = [
```

Comment:
Just my personal preference, but when exporting constant variables I prefer to name them with all caps and snake case, so it can be easily differentiated as a reusable constant, i.e. `MOCK_EVENTS_TABLE_DATA`

Reply:
I like that convention! I will make the changes :)